.circleci/config.yml

```diff
@@ -0,0 +1,76 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: docker:17.03-git
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 17.03.0-ce
+          docker_layer_caching: true
+  python_unit_test:
+    docker:
+      - image: circleci/python:2.7
+        environment:
+          DJANGO_SETTINGS_MODULE: cla_frontend.settings.testing
+    steps:
+      - checkout
+      - run:
+          name: Setup Python environment
+          command: |
+            pip install virtualenv
+            virtualenv env-ci
+      - restore_cache:
+          keys:
+            - pip-v2-{{ checksum "requirements/testing.txt" }}
+      - run:
+          name: Install dependencies
+          command: |
+            source env-ci/bin/activate
+            pip install --requirement requirements/testing.txt
+      - save_cache:
+          key: pip-v2-{{ checksum "requirements/testing.txt" }}
+          paths:
+            - "~/.cache/pip"
+      - run:
+          name: Run unit tests
+          command: |
+            source env-ci/bin/activate
+            python manage.py test --verbosity=2
+  javascript_unit_test:
+    docker:
+      - image: circleci/node:8.9.3
+        environment:
+          DJANGO_SETTINGS_MODULE: cla_frontend.settings.test
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - node_modules-{{ checksum "package-lock.json" }}
+      - run:
+          name: Install Node JS dependencies
+          command: |
+            npm install
+      - save_cache:
+          key: node_modules-{{ checksum "package-lock.json" }}
+          paths:
+            - "~/.cache/node_modules"
+      - run:
+          name: Build JavaScript application dependencies
+          command: |
+            npm run bower && npm run js
```

Comment:
If these are about building dependencies, should they be part of the "Install Node JS dependencies"?

Reply:
They're `bower` dependencies and assets that've been built by Gulp so can't go in the same dependency cache. I'll create a cache for bower, not worth it for Gulp as it's quick to build.

Reply:
Is `npm run bower` equivalent to `bower install`?